js/src/components/shipping-rate-section/minimum-order-card/groupShippingRatesByFreeShippingThreshold.test.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Internal dependencies
+ */
+import groupShippingRatesByFreeShippingThreshold from './groupShippingRatesByFreeShippingThreshold';
+
+describe( 'groupShippingRatesByFreeShippingThreshold', () => {
+	it( 'should group the shipping rates based on currency and rate', () => {
```

Comment:
Addressed in [`09af3a8` (#1445)](https://github.com/woocommerce/google-listings-and-ads/pull/1445/commits/09af3a800f069cd21914ecda061d9f9393dd80ae).

Note that I intentionally added a new test case and mentioned "rare edge case with different currencies" in the test description, so that it is separated from the common expected case.